### PR TITLE
Update 001-hello-world.md

### LINKED ISSUE #13: Following the instructions for the exercise in workshop-html-css/body/001-hello-world.md results in a failed result
The exercise is missing a step that tells the user to save the changes to the file made in the previous steps, so if you follow the instructions exactly, the "Run" button evaluates an empty file and fails. Missing step in bold below:

...
4. Surround Hello World text with opening and closing paragraph tags.
5. **Click the Save button to save your changes.**
6. Click the Run button.
...

### PR DESCRIPTION
**Pull request by getaaron**
Closes #13.

### CODE REVIEW

File: body/001-hello-world.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The most common tag is `<p>`, short for *paragraph*.
 1. Create a new file by clicking on the icon to the right of the `html-css/` directory in your code editor.
 2. Name the new file `index.html` and then click on the checkmark to save the file.
 3. Type `Hello World`.
-4. Surround `Hello World` text with opening and closing paragraph tags.
+4. Surround the `Hello World` text with opening and closing paragraph tags.
 5. Click the **Run** button.
-6. If your tests pass, click the blue "Preview My Results" button to see your application.
+6. Click the Save button to save your changes.
```
Comment (mikesjewett) [inline body/001-hello-world.md]: Save must come before Run.

File: body/001-hello-world.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ The most common tag is `<p>`, short for *paragraph*.
...
+7. If your tests pass, click the blue "Preview My Results" button to see your application.
```
Comment (mikesjewett) [inline body/001-hello-world.md]: Also, this is no longer a button. It's a tab :/

**Comment by mikesjewett** [comment]
Back to you @getaaron

File: body/001-hello-world.md (hunk 2 above)
Comment (getaaron) [inline body/001-hello-world.md]: "Documentation is a lie waiting to happen"

**Pull request merged**